### PR TITLE
[T5A2][W13-A4]Lai Hoang Dung

### PR DESCRIPTION
**Pull request by louislai**

### CODE REVIEW

**Comment by damithc** [comment]
This PR/issue should be against a repo in the org [nus-cs2103-AY1617S2](https://github.com/nus-cs2103-AY1617S2) not against the repo in se-edu org :-)
Closing...

**Comment by damithc** [comment]
My mistake.

File: src/seedu/addressbook/commands/Command.java [hunk 1]
```diff
@@ -65,4 +65,10 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    /**
+     * Check if command is a mutating comman
```
Comment (K1ang) [inline src/seedu/addressbook/commands/Command.java:70]: Take note of typo. :)

**Review by K1ang — COMMENTED** [review]
@louislai Some comments added. Please close the PR after reading comments.
Good job! Keep it up!

**Pull request closed without merging**